lib/vm.py

```diff
@@ -361,6 +365,25 @@ def test_snapshot_on_running_vm(self):
         finally:
             snapshot.destroy(verify=True)
 
+    def vdi_resize(self, vdi_uuid, new_size=None):
+        # Should MIN MAX be handled or let it throw exception?
```

Comment:
I think we should raise here.
We will soon add QCOW2 tests and it could be an opportunity to test sizes larger than 2TB. In parallel we could have a VHD specific test to check that if we exceed the limit, it does not work.

Reply:
Done